Profiler/CProfilerCallback.h

```diff
@@ -84,10 +78,10 @@ class CProfilerCallback : public CProfilerCallbackBase {
 	bool isLightMode = false;
 
 	/** 
-	 * Whether to run in eager mode and write all invocations to the trace 
-	 * file directly instead of waiting until shutdown. 
+	 * Whether to run in eager mode and write a batch of recorded invocations to the trace 

```

Comment:
please mention that the batch size is equal to the number stored here

Reply:
done